mm2src/mm2_libp2p/src/atomicdex_behaviour.rs

```diff
@@ -156,6 +160,16 @@ impl From<AdexResponse> for PeerResponse {
     }
 }
 
+#[derive(Debug)]
+pub struct ResponseOnRequestAnyRelay {
+    pub response: Option<(PeerId, Vec<u8>)>,
```

Comment:
Is it planned to add more fields to this struct? If not t might be a bit simpler to use just `Option<(PeerId, Vec<u8>)>` without wrapping into struct.

Reply:
Using `Option<(PeerId, Vec<u8>)>` maybe uninformative and makes the code more readable. Also it allows us to change the response signature without changing those function and structures where this response is just forwarded.

I guess, using `struct ResponseOnRequestAnyRelay(Option<(PeerId, Vec<u8>)>)` will be better.

Reply:
> Using Option<(PeerId, Vec<u8>)> maybe uninformative

It exists in context, e.g.:
```
RequestAnyRelay {
        req: Vec<u8>,
        response_tx: oneshot::Sender<Option<(PeerId, Vec<u8>)>>,
    }
```
And in this context it is already clear that we have the `response on request to any relay` here.

> Also it allows us to change the response signature without changing those function and structures where this response is just forwarded.

Looks like we have a response forwarding here 
https://github.com/KomodoPlatform/atomicDEX-API/pull/713/files#diff-f2aa007a2feddcd6ca151ad59debd715R192, and I guess this code will stop compiling if new field will be added to `ResponseOnRequestAnyRelay` so the goal to avoid changes in functions utilizing the struct might be not fully reached. If we want to keep the struct and reach such goal it's worth to access field directly (`response.response`) instead of destructuring (unless we explicitly want this specific code to stop compiling in case of adding new fields to `ResponseOnRequestAnyRelay`).

Reply:
Ok, I got